hackathon_site/registration/jinja2/registration/emails/waitlisted_email_body.html

```diff
@@ -0,0 +1,11 @@
+<p>Hello {{ user.first_name|striptags }},</p>
+
+<p>The {{hackathon_name}} team has reviewed your application, and due to both the high caliber and high volume of applications, we have decided to defer your application to the next round. You will hear back from us by {{final_review_response_date}}regarding a final application decision.</p>
+
+<p>By visiting your <a href="https://ieee.utoronto.ca/makeuoft/dashboard">dashboard</a>, you can join or make changes to your team, and we’ll re-consider your application along with your teammates (though due to limited spots remaining, note that we may not make the same decision on all teammates). </p>
+
+<p>If you have any questions or concerns, read the FAQ on your <a href="https://ieee.utoronto.ca/makeuoft/dashboard">dashboard</a> or feel free to contact us. </p>
```

Comment:
Ditto with above. Since you use the dashboard twice, you can define a variable for it at the top of the file like in the activation email template like:

```
{% set dashboard_url=request.build_absolute_uri(url("event:dashboard")) %}
```

then do `{{ dashboard_url }}` in the two places where you use it